manuals/1.0/en/apidoc.md

```
@@ -4,177 +4,118 @@ title: API Doc
+| `php-version` | `'8.2'` | PHP version |
+| `format` | `'html,openapi'` | Comma-separated: html, md, openapi, alps |
+| `alps-profile` | `''` | ALPS profile path (required for alps format) |
+| `docs-path` | `'docs/api'` | Output directory |
+| `publish-to` | `'github-pages'` | `github-pages` or `artifact-only` |
 
-* If there is no `@param` description in the phpdoc of the method, get the information of the argument from the method signature.
-* The order of priority for information acquisition is phpdoc, JSON schema, and profile.
+### Output Structure
 
-## Configuration
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)